openwisp_radius/models.py

```diff
@@ -88,4 +88,4 @@ class Meta(AbstractRadiusUserProfile.Meta):
         swappable = swappable_setting('openwisp_radius', 'RadiusUserProfile')
 
```

Comment:
Unnecessary lines @Mah1ndra?

Reply:
Also remove the first line importing get_user_model

Reply:
okay. Thank you for the review @rohithasrk